Form/FormMapper.php

```diff
@@ -301,4 +301,12 @@ protected function setTabs(array $tabs)
     {
         $this->admin->setFormTabs($tabs);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName()
+    {
+        return 'form';
```

Comment:
This is not forward-compatible, you have to return a class name for the last versions of Symfony.